api/lib/infrastructure/jobs/cpf-export/handlers/send-email.js

```diff
@@ -1,20 +1,14 @@
 import { config } from '../../../../config.js';
-import cronParser from 'cron-parser';
 import { logger } from '../../../logger.js';
 
 const { cpf } = config;
 const sendEmail = async function ({ getPreSignedUrls, mailService }) {
-  const lastGeneratedFilesDate = cronParser
-    .parseExpression(cpf.plannerJob.cron, { tz: 'Europe/Paris' })
-    .prev()
-    .toDate();
-
-  const generatedFiles = await getPreSignedUrls({ date: lastGeneratedFilesDate });
+  const generatedFiles = await getPreSignedUrls();
 
   if (generatedFiles.length) {
     await mailService.sendCpfEmail({ email: cpf.sendEmailJob.recipient, generatedFiles });
   } else {
-    logger.info(`No new generated cpf files since ${lastGeneratedFilesDate}`);
+    logger.info(`No new generated cpf files`);
```

Comment:
Du fait comme nous avons pris le parti de se dire que la création et l'envoi d'email sont des processus indépendants l'un de l'autre, du coup ce log ne doit pas porter de notion de "est-ce que des fichiers ont été générés ou non".

```suggestion
    logger.info(`No CPF exports files ready to send`);
```

Reply:
corrigé